unicon_backend/models/organisation.py

```diff
@@ -44,6 +47,48 @@ class Role(RoleBase, table=True):
     id: int | None = Field(default=None, primary_key=True)
     project_id: int = Field(foreign_key="project.id")
 
+    # Assignable permissions
+    view_problems_access: bool = Field(
+        default=False,
+        sa_column=sa.Column(sa.Boolean(), server_default="0"),
```

Comment:
is `sa_column` needed here? iirc, `SQLModel::Field` is directly compatible (like how `id` and `project_id` fields are defined above)

Reply:
not anymore - added server_default after running into problems with multiple migrations, but that is now solved by regeneration 👍

Reply:
oh i stand corrected i can't fully remove this since we are splitting it into 2 migrations 
will replace with `sa_column_kwargs={"server_default": "false"}` so i am at least not using sa.Boolean()